server/src/main/scala/services/graphstages/ArrivalsGraphStage.scala

```diff
@@ -272,8 +290,14 @@ class ArrivalsGraphStage(name: String = "",
             .arrivalDataSanitiserWithoutThresholds
             .withSaneEstimates(mergedLiveArrival)
           Option(sanitisedLiveArrival)
+        case (None, Some(baseLiveArrival)) if scheduleTimeApproximation(key, baseLiveArrival,true).isDefined => {
```

Comment:
is this match correct? baseLiveArrival is the Cirium Feed Arrival - in this case we've actually already found a direct match. 

Shouldn't we be looking at the case `(Some(liveArrival), None) if scheduleTimeApproximation(key, liveArrival) =>`

for this fix?